packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -0,0 +1,315 @@
+            <img
+                :class="[
+                    $style['c-stampCard1-icon']
+                ]"
+                :data-test-id="testIdForSection('image')"
+                :src="image"
+                :alt="title"
+            >
+            <h3
+                :class="[
+                    $style['c-stampCard1-title']
+                ]"
+                :data-test-id="testIdForSection('title')"
+            >
+                {{ title }}
+            </h3>
+
+            <p
+                :class="[
+                    $style['c-stampCard1-statusText']
+                ]"
+                :data-test-id="testIdForSection('statusText')"
+                v-html="statusText"
+            />
+        </div>
+        <div
+            v-if="isReadyToClaim"
+            :class="[
+                $style['c-stampCard1-redemptionDetails']
+            ]"
+            :data-test-id="testIdForSection('redemptionDetails')"
+        >
+            <div
+                v-for="(subStatusLine, index) in subStatusText"
+                :key="index"
+                :class="[
+                    $style['c-stampCard1-subStatusText']
+                ]"
+                :data-test-id="testIdForSection('subStatusText', index)"
+            >
+                {{ subStatusLine }}
+            </div>
+            <div
+                :class="[
+                    $style['c-stampCard1-expiryInfo']
+                ]"
+                :data-test-id="testIdForSection('expiryInfo')"
+            >
+                {{ expiryLine }}
+            </div>
+        </div>
+        <div
+            v-else
+            :class="[
+                $style['c-stampCard1-stamps']
+            ]"
+            :data-test-id="testIdForSection('stamps')"
+        >
+            <div
+                v-for="({ stampImage, classSuffix }, index) in stamps"
+                :key="index"
+            >
+                <div
+                    :class="[
+                        $style[`c-stampCard1-stamp`]
+                    ]"
+                    :data-test-id="testIdForSection(`stamp${classSuffix}`)"
+                >
+                    <component
+                        :is="stampImage"
+                        :class="[
+                            $style[`c-stampCard1-stampImage`],
+                            $style[`c-stampCard1-stamp${classSuffix}`]
+                        ]"
+                    />
+                </div>
+            </div>
+        </div>
+    </component>
+</template>
+
+<script>
+import EmptyStamp from './images/stamp-empty-15.svg';
+import FullStamp from './images/stamp-full-15.svg';
+
+export default {
+    name: 'StampCard1',
+
+    components: {
+        EmptyStamp,
+        FullStamp
+    },
+
+    props: {
+        card: {
+            type: Object,
+            required: true
+        },
+
+        testId: {
+            type: String,
+            default: null
+        }
+    },
+
+    computed: {
+        title () {
```

Comment:
I think there is no benefit here in using computed properties for shortening prop names. `this.card.title`, `this.card.description`, `this.card.image`, `this.card.url`, `this.card.discountPercentage`, `this.card.isReadyToClaim` etc should be fine.

Reply:
The purpose of having these is not only to shorten prop names, but also as a clue (when changing the template, or stories) as to what keys *are* and *should* be available in the incoming card object. I'd prefer to leave it in if possible.

Reply:
Sorry, I still don't see the benefit. If you want to leave a clue for the object keys than it is better to do in props (via validator function)

Reply:
Here is [an example](https://stackoverflow.com/questions/43913556/nested-property-types-in-vue-js) of what I mean. In React there is "object of shape" for this purpose but I couldn't find an alternative in Vue closer that validator.

Reply:
Or another way is to document this in README :)

Reply:
Thanks for the reasoning - in my opinion the validator function in props is the wrong place to do it in this instance as we need to be fairly widely accepting of data provided by braze - some keys may be specified while others may be not, and we should do our best effort to show the card based on the data we _do_ have available, where a validator function would reject outright.

The README however, I can get behind, I'll make this change.